CDP4Dal/ISession.cs

```diff
@@ -312,20 +312,13 @@ public interface ISession
         /// </returns>
         Task CloseModelRdl(ModelReferenceDataLibrary modelRdl);
 
-        /// <summary>
-        /// Queries the current <see cref="DomainOfExpertise"/> from the session for the current <see cref="Iteration"/>
-        /// </summary>
-        /// <returns>
-        /// The <see cref="DomainOfExpertise"/> if selected, null otherwise.
-        /// </returns>
-        DomainOfExpertise QueryCurrentDomainOfExpertise();
-
         /// <summary>
         /// Queries the <see cref="Participant"/>'s <see cref="DomainOfExpertise"/>'s from the session for the current <see cref="Iteration"/>
         /// </summary>
+        /// <param name="iteration">The <see cref="Iteration"/></param>
         /// <returns>
         /// The <see cref="DomainOfExpertise"/> if selected, null otherwise.
         /// </returns>
-        IEnumerable<DomainOfExpertise> QueryDomainOfExpertise();
+        IEnumerable<DomainOfExpertise> QueryDomainOfExpertise(Iteration iteration);

```

Comment:
in the source file, can this method be placed after the existing `QuerySelectedDomainOfExpertise`, then they are close to eachother, should be easier to maintain the code in the future.